releasenotes/notes/remove-deprecated-in-quantum-info-0f9bd2b0c093307d.yaml

```
@@ -0,0 +1,6 @@
+---
+upgrade:
+  - |
+    The class ``qiskit.quantum_info.PauliTable`` and `qiskit.quantum_info.StabilizerTable`
+    is removed. The function ``random_pauli_table`` is also removed.
+    They were deprecated in Qiskit 0.43 (with Terra 0.24), released on May 2023.
```

Could we just add a brief sentence here mentioning what the alternatives are (`PauliList` and `random_pauli_list`, I think?)?

Something like this, but feel free to use your own wording.
```suggestion
    The classes ``qiskit.quantum_info.PauliTable`` and `qiskit.quantum_info.StabilizerTable`
    are removed. The function ``random_pauli_table`` is also removed.
    They were deprecated in Qiskit 0.43 (with Terra 0.24), released in May 2023.
    Instead, you should use :class:`.PauliList` and :func:`.random_pauli_list`.
```

---

(note that I had a couple of minor grammar tweaks in the suggestion as well as the extra sentence - I was expecting GitHub to highlight them in the diff, but it doesn't seem to have.)

---

Thank you very much. Yes, `PauliList` and `random_pauli_list` is the alternative.